### PR TITLE
Update iOS SDK documentation.

### PR DESCRIPTION
**Pull request by IhmeHippi**
iOS SDK now supports SwiftPM. Added instructions for installation through SwiftPM.

### CODE REVIEW

File: modules-sdks/mobile-sdk/ios.md [hunk 1]
```diff
@@ -428,6 +449,8 @@ sequenceDiagram
                          next_href="custom-backend"
                          next_title="Next: Custom Backend" %}
 
+[xcode-swiftpm]: https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app
+[sdk-package-repo]: https://github.com/SwedbankPay/swedbank-pay-sdk-ios.git
 [sdk-pod]: https://cocoapods.org/pods/SwedbankPaySDK
 [cocoapods]: https://cocoapods.org/
 [cocoapods-gettingstarted]: https://guides.cocoapods.org/using/getting-started.html
```
Comment (asbjornu) [inline modules-sdks/mobile-sdk/ios.md:456]: Seems like some of these link references are no longer in use?

```
modules-sdks/mobile-sdk/ios.md
  456:1-456:84  warning  Found unused definition  no-unused-definitions  remark-lint
  457:1-457:75  warning  Found unused definition  no-unused-definitions  remark-lint
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
This is weird. GitHub thinks the PR is out of date, but I can't update it?

image

**Comment by IhmeHippi** [comment]
Should I manually rebase it against develop and force push?

**Comment by asbjornu** [comment]
Yes, please do.

**Comment by IhmeHippi** [comment]
Rebased

**Pull request merged**